target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-rax120.dts

```diff
@@ -0,0 +1,1010 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/dts-v1/;
+/* Copyright (c) 2017, The Linux Foundation. All rights reserved.
```

Comment:
Better put your copyright of remove this, I assume this is copy/paste from QCA

Reply:
Well it started as a copy from hk01 from upstream. Restructured manually using QCA and other random knowledge from all over the place.

Reply:
That upstream HK01 DTS is a dumpster fire, and the Linux Foundation copyright makes no sense, better just remove it.

Reply:
Maybe I can make it a single line "// SPDX-License-Identifier: GPL-2.0-or-later OR MIT"?

No rights reserved on my end.

Reply:
License itself is fine, just remove the copyright